web/src/pages/Register/SignUp.tsx

```diff
@@ -0,0 +1,148 @@
+import "./signup.scss";
```

Comment:
You should remove this, this directory is already pushed in current under a new name